joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/dataflow/ArrayDataflowTests.scala

```diff
@@ -0,0 +1,24 @@
+package io.joern.go2cpg.dataflow
+import io.shiftleft.semanticcpg.language._
+import io.joern.dataflowengineoss.language._
+import io.joern.go2cpg.testfixtures.GoCodeToCpgSuite
+
+class ArrayDataflowTests extends GoCodeToCpgSuite {
```

Comment:
For dataflows test, you need to pass the `withDataflow` flag in GoCodeToCpgSuite 🤔

Reply:
My bad again. Works now.